scripts/feature_toggle_report.py

```diff
@@ -68,7 +68,7 @@ def main(annotations_dir, toggle_data_dir, output_file_path, env, toggle_type, v
 
     # process configuration
     # commandline-option inputs overwrite configuration file data
-    toggle_type_filter = toggle_type
+    toggle_type_filter = [ToggleTypes.get_internally_consistent_toggle_type(t_type) for t_type in toggle_type]
```

Comment:
The `--toggle-type` argument is defined with `default=None`, and I think that will cause a problem for this iteration. Maybe the `default=None` is ignored? But either way, it should probably be removed.